crates/nu-command/src/conversions/fmt.rs

```diff
@@ -131,6 +132,40 @@ fn fmt_it(num: i64, span: Span) -> Value {
     Value::Record { cols, vals, span }
 }
 
+fn fmt_it_64(num: f64, span: Span) -> Value {
+    let mut cols = vec![];
+    let mut vals = vec![];
+
+    cols.push("binary".into());
+    vals.push(Value::string(format!("{:b}", num.to_bits()), span));
+
+    cols.push("debug".into());
+    vals.push(Value::string(format!("{num:#?}"), span));
+
+    cols.push("display".into());
+    vals.push(Value::string(format!("{num}"), span));
+
+    cols.push("lowerexp".into());
+    vals.push(Value::string(format!("{num:#e}"), span));
+
+    cols.push("lowerhex".into());
+    vals.push(Value::string(format!("{:x}", num.to_bits()), span));
+
+    cols.push("octal".into());
+    vals.push(Value::string(format!("{:o}", num.to_bits()), span));
+
+    // cols.push("pointer".into());
+    // vals.push(Value::string(format!("{:#p}", &num), span));
+
+    cols.push("upperexp".into());
+    vals.push(Value::string(format!("{num:#E}"), span));
+
+    cols.push("upperhex".into());
+    vals.push(Value::string(format!("{:X}", num.to_bits()), span));
```

Comment:
upperhex should have `0X` prefix, lowerhex `0x` and octal should have `0o` prefix, so they are well-formed Nu numeric literals.

Reply:
should have been fixed in https://github.com/nushell/nushell/pull/9142/commits/c22c7f4d7de227d808214d75409d460e65c4e3a9 @bobhy :relieved:

Reply:
@amtoine
You mean have to revert it ?

Reply:
no no that's perfect what you've done :blush: 
as i was there, i wanted to point out the commit for @bobhy to see the changes directly :wink: